packages/neuron-wallet/src/controllers/wallets/index.ts

```diff
@@ -331,7 +331,13 @@ export default class WalletsController {
 
   @CatchControllerError
   public static async updateAddressDescription({ address, description }: { address: string; description: string }) {
-    // TODO: update description of specified address
+    const walletService = WalletsService.getInstance()
+    const wallet = walletService.getCurrent()
```

Comment:
Please refactor that: 1) wallet id is always passed to controllers or other wallet layers from UI, 2) `neuron-wallet` uses `walletService.getCurrent()` as few as possible.

Reply:
updated